testSrc/unit/io/flutter/logging/util/LineParserTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging.util;
+
+import com.intellij.execution.filters.BrowserHyperlinkInfo;
+import com.intellij.execution.filters.UrlFilter;
+import com.intellij.ui.SimpleTextAttributes;
+import com.intellij.util.ReflectionUtil;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.awt.*;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings("UseJBColor")
```

Comment:
This suppress warnings about using JBColor brings up an interesting question. Are there reasonable ways to use JBColor here so that the colored output looks good in both the dark and light themes?

Reply:
I wondered that too and have it on my list to discuss w/ @InMatrix.  As it stands, the way colors are defined using ansi codes is a little low-level and doesn't map naturally to theming.  Definitely something to consider.

Reply:
FWIW: I just re-checked and I had already fixed things to not require this suppression so it's now removed.  The general issue still stands though.  Thanks for bringing it up!